### PR TITLE
Use different animation depending on Shell Side Arm category.

### PR DESCRIPTION
**Pull request by Karthik99999**
Uses Sludge Bomb for special, Poison Jab for physical. I left the hints in for accessibility reasons, but I can remove it if you guys want me to.

### CODE REVIEW

**Comment by DaWoblefet** [comment]
I like this!

**Comment by xInstruct** [comment]
This is an issue in Staff Bros, so this is 100% gonna come up. There's gonna be false positives where the animation would hit but would actually miss.

**Comment by Karthik99999** [comment]
> This is an issue in Staff Bros, so this is 100% gonna come up. There's gonna be false positives where the animation would hit but the move itself would actually miss.

Well this is a problem with showdown in general, cause move animations just move a little bit to the side if it misses instead of just not playing. Though I guess this is the one circumstance where this showdown behavior matters.

File: data/moves.ts [hunk 1]
```diff
@@ -15121,6 +15121,18 @@ export const Moves: {[moveid: string]: MoveData} = {
 		pp: 10,
 		priority: 0,
 		flags: {protect: 1, mirror: 1},
+		onTryMove() {
+			this.attrLastMove('[still]');
```
Comment (Zarel) [inline data/moves.ts:15125]: I'm a bit uncomfortable with the animation removal and the animation adding to be on two separate events. Why aren't these both in `onPrepareHit`?

Also `attrLastMove('[anim] Poison Jab')` works, right?

Comment (Zarel) [reply data/moves.ts:15125]: Also, may want to `attrLastMove('[anim] Shell Side Arm Physical')` and then alias it to Poison Jab in `move-animations`.

Comment (xInstruct) [reply data/moves.ts:15125]: > attrLastMove('[anim] Shell Side Arm Physical')

wait thats a thing????

Comment (Karthik99999) [reply data/moves.ts:15125]: I just copied the staff bros behavior tbh. And yes `attrLastMove('[anim] Poison Jab')` does work.

Comment (Zarel) [reply data/moves.ts]: @xInstruct lol yes Staff Bros should probably be using it, too.

**Comment by Zarel** [comment]
ty!

**Pull request merged**